docs/docs/concepts/retrieval_unstructured.mdx

```diff
@@ -0,0 +1,160 @@
+# Retrieval for Unstructured Data
+
+## Motivation
+
+Language models (LLMs) are trained on vast but fixed datasets, which limits their ability to access up-to-date or domain-specific information. 
+
+To enhance their performance on specific tasks, we can augment their knowledge using retrieval systems.
```

Comment:
`retrieval system` is one technique for increasing performance on specific task (another technique is just prompting).

Perhaps re-sequence here to something like this:

1. Can enhance performance on specific task by presenting the models with relevant information as part of their prompt
2. One technique to do so is by retrieving relevant information using a search engine.